tests/bench/explicit.py

```
@@ -0,0 +1,1184 @@
+- GOOD: Minor issues, usable
+- MARGINAL: May cause ambiguity
+- POOR: Significant issues
+- UNUSABLE: Cannot serve as premise"""
+
+
+# =============================================================================
+# JUDGE IMPLEMENTATION
+# =============================================================================
+
+class ExplicitJudge:
+    def __init__(
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)